src/types/typeobject.rs

```diff
@@ -127,6 +130,16 @@ pub trait PyTypeMethods<'py>: crate::sealed::Sealed {
     fn is_subclass_of<T>(&self) -> PyResult<bool>
     where
         T: PyTypeInfo;
+
+    /// Return Python mro
+    ///
+    /// Equivalent to the Python expression `__mro__`.
```

Comment:
```suggestion
    /// Return the method resolution order for this type.
    ///
    /// Equivalent to the Python expression `self.__mro__`.
```

I wonder if it's worth linking to https://docs.python.org/3/library/stdtypes.html#class.__mro__ ?